index.bs

```diff
@@ -5946,6 +5950,110 @@ The [=remote end steps=] with |command parameters| are:
 
 </div>
 
+#### The emulation.setScriptingEnabled Command ####  {#command-emulation-setScriptingEnabled}
+
+The <dfn export for=commands>emulation.setScriptingEnabled</dfn> command emulates
+disabling JavaScript on web pages.
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+    <pre class="cddl" data-cddl-module="remote-cddl">
+      emulation.SetScriptingEnabled = (
+        method: "emulation.setScriptingEnabled",
```

Comment:
Method name aligns with HTML's ["Scripting is enabled"](https://html.spec.whatwg.org/multipage/webappapis.html#concept-environment-script).